internal/conf/configuration.go

```diff
@@ -217,6 +218,12 @@ type TextlocalProviderConfiguration struct {
 	Sender string `json:"sender" split_words:"true"`
 }
 
+type Msg91ProviderConfiguration struct {
+	AuthKey       string `json:"auth_key" split_words:"true"`
+	SenderId      string `json:"sender_id" split_words:"true"`
+	DltTemplateId string `json:"dlt_template_id" split_words:"true"`
```

Comment:
i'm not familiar with what a DltTemplateId is - can you please link the relevant documentation for this? is it a mandatory field or can you send messages through msg91 without it?